Features/Context/SubContext/Fields.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/**
+ * File containing the Common Functions for context class PlatformUI.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context\SubContext;
+
+use EzSystems\BehatBundle\ObjectManager\FieldType;
+use Behat\Mink\WebAssert;
+
+trait Fields
+{
+    /**
+     * @When I create a content of this type
```

Comment:
@andrerom That is right. This sentence is to be used in conjunction with the other ones for creating contenttypes in way that is easier to read/understand. These sentences are more focused on the fieldtypes so they provide an abstraction for creating contenttypes and content.